metaflow/client/core.py

```diff
@@ -759,7 +759,8 @@ class MetaflowCode(object):
 
     You can extract code in the directory `snapshot` like so:
     ```
-    Run(...).code.tarball.extractall(path='snapshot')
+    from metaflow.util import tar_safe_extract
```

Comment:
is this needed? we don't expose any util methods from metaflow.util to the end user. instead we can recommend any method that the user has at their disposal to do tar extraction.

Reply:
Sure. I can remove the entire comment and just say that the tarball is given by Run(...).code.tarball and just leave it at that. I didn't want to recommend something that is potentially vulnerable (I know the risk is very very small).

Reply:
That works!

Reply:
Please merge the PR after that change is in :)